src/main/java/com/github/havardh/javaflow/exceptions/AggregatedException.java

```diff
@@ -0,0 +1,18 @@
+package com.github.havardh.javaflow.exceptions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AggregatedException extends RuntimeException {
```

Comment:
Would be nice with some unit tests on this class to verify the message output when the exception is thrown.